src/lib/study-webidl.js

```diff
@@ -236,39 +236,39 @@ function studyWebIdl (specs, { crawledResults = [], curatedResults = [] } = {})
     if (ifaceExposure.length === 0 && !iface.partial) {
-      recordAnomaly(spec, 'noExposure', `The interface "${iface.name}" has no [Exposed] extended attribute`);
+      recordAnomaly(spec, 'noExposure', `The interface \`${iface.name}\` has no \`[Exposed]\` extended attribute`);
     } else if (ifaceExposure.length > 0 && ifaceExposure[0] !== '*') {
       // Make sure that the interface is exposed on known globals
       const unknown = ifaceExposure.filter(e => !Object.keys(globals).includes(e));
       if (unknown.length > 0) {
-        recordAnomaly(spec, 'unknownExposure', `The [Exposed] extended attribute of the interface "${iface.name}" references unknown global(s): ${unknown.join(', ')}`);
+        recordAnomaly(spec, 'unknownExposure', `The \`[Exposed]\` extended attribute of the interface \`${iface.name}\` references unknown global(s): ${unknown.join(', ')}`);
       }
     }
 
     // Make sure that exposure of partial is compatible with original
     if (iface.partial) {
       const mainExposure = getExposure(mainInterface);
       if (ifaceExposure[0] === '*' && mainExposure[0] !== '*') {
-        recordAnomaly(spec, 'incompatiblePartialIdlExposure', `The partial interface "${iface.name}" is exposed on all globals but the original interface is not (${mainExposure.join(', ')})`);
+        recordAnomaly(spec, 'incompatiblePartialIdlExposure', `The partial interface \`${iface.name}\` is exposed on all globals but the original interface is not (${mainExposure.join(', ')})`);
       } else if (ifaceExposure.length > 0 && mainExposure.length > 0 && mainExposure[0] !== '*') {
         const problematic = ifaceExposure.filter(x => !mainExposure.includes(x));
         if (problematic.length > 0) {
-          recordAnomaly(spec, 'incompatiblePartialIdlExposure', `The [Exposed] extended attribute of the partial interface "${iface.name}" references globals on which the original interface is not exposed: ${problematic.join(', ')} (original exposure: ${mainExposure.join(', ')})`);
+          recordAnomaly(spec, 'incompatiblePartialIdlExposure', `The \`[Exposed]\` extended attribute of the partial interface \`${iface.name}\` references globals on which the original interface is not exposed: ${problematic.join(', ')} (original exposure: ${mainExposure.join(', ')})`);
         }
       }
     }
   }
 
   function checkEnumMultipleValues (spec, idl) {
     if (idl.values.length <= 1) {
-      recordAnomaly(spec, 'singleEnumValue', `The enum "${idl.name}" has fewer than 2 possible values`);
+      recordAnomaly(spec, 'singleEnumValue', `The enum \`${idl.name}\` has fewer than 2 possible values`);
     }
   }
 
   function checkSyntaxOfEnumValue (spec, idl) {
     idl.values.forEach(({ value }) => {
-      if (value.match(/[A-Z_]/)) {
-        recordAnomaly(spec, 'wrongCaseEnumValue', `The value "${value}" of the enum "${idl.name}" does not match the expected conventions (lower case, hyphen separated words)`);
+      if (value.match(/[A-Z_\s]/)) {
```

Comment:
That's the only substantive change in this pull request!
The rest is just backticks updates.